cli/cmd/plugin/ui/web/tanzu-ui/src/components/aws/AwsWizard.tsx

```diff
@@ -0,0 +1,25 @@
+// React imports
```

Comment:
This component feels like it should be more configurable and reusable. Going back to Shimon's thought, could we have AwsManagementCluster pass in props/a callback to tell this wizard what part of the state it should be aware of, what components it should include, etc.

@swalner-vmware do you have any thoughts on this?

Reply:
I'd be fine simply renaming this one to AwsMCCreateSimple and then having another AwsMCCreateAdvanced that reuses a bunch of the same components, but adds some new ones.

Reply:
Due to library limitation, the step components are hard to config.